src/index.js

```diff
@@ -0,0 +1,8 @@
+import * as tasks from './tasks/index';
```

Comment:
It is good practice to import explicitly only necessary object, for readability and for the IDE.

Reply:
;) One more argument - if you use more import * you can pollute namespace and there is risk of a conflcting names